pkg/tf2pulumi/convert/testdata/mappings/maxItemsOne.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "maxItemsOne",
+  "provider": {
+    "resources": {
```

Comment:
We should probably check this is correct for data sources as well.
Plus it would be good to cleanup whatever is currently testing the "maxItems=1" logic and cover that here so this mapping gives us coverage of "maxItems=1 && maxItemsOne=false" + just "maxItems=1".

Reply:
> Plus it would be good to cleanup whatever is currently testing the "maxItems=1" logic and cover that here so this mapping gives us coverage of "maxItems=1 && maxItemsOne=false" + just "maxItems=1".

Already did that in `convertBody` 😄 `isList := !scopes.maxItemsOne(blockPath)`

> We should probably check this is correct for data sources as well.

Will add a test case for it ✅